modules/openapi-generator/src/test/java/org/openapitools/codegen/typescript/typescriptangular/TypeScriptAngularClientCodegenTest.java

```diff
@@ -464,4 +464,29 @@ public void testEnumAsConst() throws IOException {
         assertThat(fileContents).containsOnlyOnce("} as const;");
         assertThat(fileContents).doesNotContain(" as Type");
     }
+
+    @Test
+    public void testDeepObject() throws IOException {
+        // GIVEN
+        final String specPath = "src/test/resources/3_0/deepobject.yaml";
+
+        File output = Files.createTempDirectory("test").toFile();
+        output.deleteOnExit();
+
+        // WHEN
+        final CodegenConfigurator configurator = new CodegenConfigurator()
+            .setGeneratorName("typescript-angular")
+            .setInputSpec(specPath)
+            .setOutputDir(output.getAbsolutePath().replace("\\", "/"));
+
+        final ClientOptInput clientOptInput = configurator.toClientOptInput();
+
+        Generator generator = new DefaultGenerator();
+        generator.opts(clientOptInput).generate();
+
+        // THEN
+        final String fileContents = Files.readString(Paths.get(output + "/api/default.service.ts"));
+        assertThat(fileContents).containsOnlyOnce("<any>options, 'options', true);");
```

Comment:
are these assertions related to the deepObject case?

Reply:
Yes, the last parameter is `isDeep`.